using-mobile-extensions/adobe-target/adobe-target-release-notes.md

```diff
@@ -20,6 +20,8 @@
 
 * Fixed an issue where the click notification was not being sent to Adobe Target for a retrieved mbox location upon the `clickedLocation` API call.
 
+* Added support for sending click conversion A4T payload to Adobe Analytics for A4T enabled Target activities upon the `clickedLocation` API call.
```

Comment:
"... for sending the click conversion..." 

"... for A4T-enabled Target activities when the `clickLocation` API is called."